cert/CACreationScript.sh

```diff
@@ -108,25 +108,33 @@ openssl ca -create_serial -cert $PI.crt -keyfile $PI.pkey -outdir $CSR -batch -o
 #PAL: TODO
 
 #Device Intermediate
-CBSD=wif_cbsd_ca
+CBSD=cbsd_ca
 CBSD_SIGN=$CBSD$SIG
 
-openssl req -new -newkey rsa:$INT_RSA_KEY_SIZE -nodes -reqexts $CBSD -out $CBSD.csr -keyout $CBSD.pkey -subj "/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=WInnForum RSA Device CA-1" -config openssl.cnf
+openssl req -new -newkey rsa:$INT_RSA_KEY_SIZE -nodes -reqexts $CBSD -out $CBSD.csr -keyout $CBSD.pkey -subj "/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=WInnForum RSA CBSD CA-1" -config openssl.cnf
 
 openssl ca -create_serial -cert $ROOT.crt -keyfile $ROOT.pkey -outdir $CSR -batch -out $CBSD.crt -utf8 -days $INT_VALIDITY -policy policy_anything -md sha384 -extensions $CBSD_SIGN -config openssl.cnf -in $CBSD.csr
 
 #Device EE
-CBSD_EE=wif_cbsd_req
+CBSD_EE=cbsd_req
 CBSD_EE_SIGN=$CBSD_EE$SIG
 
-openssl req -new -newkey rsa:$EE_RSA_KEY_SIZE -nodes -reqexts $CBSD_EE -out $CBSD_EE.csr -keyout $CBSD_EE.pkey -subj "/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=Device End-Entity Example" -config openssl.cnf
+openssl req -new -newkey rsa:$EE_RSA_KEY_SIZE -nodes -reqexts $CBSD_EE -out $CBSD_EE.csr -keyout $CBSD_EE.pkey -subj "/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=CBSD End-Entity Example" -config openssl.cnf
 
 openssl ca -create_serial -cert $CBSD.crt -keyfile $CBSD.pkey -outdir $CSR -batch -out $CBSD_EE.crt -utf8 -days $EE_VALIDITY -policy policy_anything -md sha384 -extensions $CBSD_EE_SIGN -config openssl.cnf -in $CBSD_EE.csr
 
-#TODO: manufacturer intermediate signed by $CBSD, and EE cert signed by manufacturer intermediate
-
+#Device OEM Intermediate
```

Comment:
I think that's how I have it configured currently.
CBSD_CA is issued from the ROOT and has a pathlength of 1. It can issue either CBSD_EE certs or CBSD_OEM_CA certs.
CBSD_OEM_CA is issued from CBSD_CA and has a pathlength of 0. It can issue only CBSD_OEM_EE certs (which are functionally equivalent to CBSD_EE certs).
So, for devices, there's a common intermediate that can either directly issue Device certs _or_ issue an additional signing intermediate so OEMs can issue their own Device certs.